tests/testthat/test-submit-model.R

```diff
@@ -4,16 +4,19 @@ context("submit_model(.dry_run=T)")
 # testing single model submission
 ###################################
 
-# create fake bbi.yaml
-readr::write_file("created_by: test-submit-model", "bbi.yaml")
-on.exit({ fs::file_delete("bbi.yaml")})
+
 
 model_dir <- ABS_MODEL_DIR
 mod_ctl_path <- file.path(model_dir, CTL_FILENAME)
 
+# create fake bbi.yaml
+readr::write_file("created_by: test-submit-model", file.path(model_dir, "bbi.yaml"))
+on.exit({ fs::file_delete(file.path(model_dir, "bbi.yaml"))})
```

Comment:
I like this fake bbi.yaml approach better than the `bbi_init` calls in test-print.R and test-submit-models.R that came from my patch at https://github.com/metrumresearchgroup/bbr/pull/496#discussion_r900086187

For consistency, I think it'd make sense to convert those to using this fake bbi.yaml approach, though no strong objections if you want to leave it as is.

Reply:
Yeah I agree. I noticed that that file didnt have any fake bbi.yaml's in there before, so wasn't sure if models were running in various directories (didnt spend as long looking through it). Will make that change assuming that isnt the case though